tests/integration/test_katib_experiments.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Integration tests for Katib Experiments.
+"""
+
+import logging
+
+import lightkube
+import pytest
+from pytest_operator.plugin import OpsTest
+from utils import (
+    assert_deleted,
+    assert_exp_status_running_succeeded,
+    assert_get_experiment,
+    create_experiment,
+    delete_experiment,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize(
+    "experiment_file",
+    [
+        "tests/assets/crs/bayesian-optimization.yaml",
+        "tests/assets/crs/cmaes.yaml",
+        "tests/assets/crs/darts-cpu.yaml",
+        "tests/assets/crs/enas-cpu.yaml",
+        "tests/assets/crs/file-metrics-collector.yaml",
+        "tests/assets/crs/grid-example.yaml",
+        "tests/assets/crs/hyperband.yaml",
+        "tests/assets/crs/median-stop.yaml",
+        "tests/assets/crs/random.yaml",
+        "tests/assets/crs/simple-pbt.yaml",
+    ],
+)
```

Comment:
I'm not sure if you guys have already had any discussion about this, but do you think it would make sense to have automatic CR discovery here, instead of explicitly passing every path? I believe it would be great, given that these all reside in the `assets/crs` directory. You can see how I implemented the exact same thing for the UATs here: https://github.com/canonical/charmed-kubeflow-uats/blob/main/tests/test_notebooks.py

Reply:
+1 to @phoevos' suggestion. We do something very similar in training-operator: https://github.com/canonical/training-operator/blob/main/tests/integration/test_charm.py#L70

Reply:
Wow, I'd never seen `glob` before, cool!